adafruit_pybadger/pybadger_base.py

```diff
@@ -447,6 +453,8 @@ def show_business_card(
         name_string=None,
         name_scale=1,
         name_font=terminalio.FONT,
+        font_color=0xFFFFFF,
+        background_color=None,
```

Comment:
can we use `font_background_color` instead of `background_color` for the name of this argument? Without "font" in the name of the argument I'm afraid folks might get confused a little bit expecting it to set the background color of the full business card (if their image doesn't cover the full screen) rather than specifically the background behind the text..

Reply:
@FoamyGuy that makes sense.  I should have put that thought into.  Thanks!  I am glad pylint passed, it wasn't on my Windows workstation so I was a little nervous.  Well I will get working on that, if I don't finish before the weekend, Have a great Weekend!

Reply:
@FoamyGuy I think I messed up.  I was trying out VSCode and its git built-ins.  So now I have created a .vscode in my branch, oops!  Yikes!  I am no longer compliant.  I was able to remove from local repository (on computer) but it still is on Github.com repository.  Not sure how to remove.

Reply:
No worries. I use PyCharm and sometimes have a similar issue with it trying to add things to my repos on accident. I suspect there is a way to remove them from the repo using the `git` commandline tool (and probably some way within vscode itself too, but I'm not familiar with that). 

I don't know the exact right command though and I'd hate to guess wrong and send you further down the rabbit hole. 

I will try to remove it and make a new commit for you. We could probably add it it to the gitignore file as well so that it will know not to add them in the future for anyone that is using vscode.